static/app/views/issueList/issueListSetAsDefault.spec.tsx

```diff
@@ -72,4 +81,30 @@ describe('IssueListSetAsDefault', () => {
       );
     });
   });
+
+  it('does not render anything when on default search and no pinned search', () => {
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/searches/',
+      body: [TestStubs.Search({isPinned: false, query: 'browser:firefox'})],
+    });
+
+    render(<IssueListSetAsDefault {...defaultProps} query="is:unresolved" />, {
+      organization,
+    });
+
+    expect(screen.queryByRole('button')).not.toBeInTheDocument();
```

Comment:
@scttcper pointed out that you can use expect(container).toBeEmptyDOMElement();` for this type of thing too

Reply:
That is a more accurate way to test this, but I've been avoiding using that because if we make `render` render modals (or anything else) by default that line will fail

Reply:
Ah hmmm. Thoughts @scttcper

Reply:
i only use it when the entire container is returning null